scripts/create-mysql.sh

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 
-cat > /home/vagrant/.my.cnf << EOF
+cat > /root/.my.cnf << EOF
 [client]
 user = homestead
 password = secret
 host = localhost
 EOF
 
+cp /root/.my.cnf /home/vagrant/.my.cnf
```

Comment:
I'm curious what the file permissions for this is after creating it this way. 🤔 Any issues with `mysq`l reading it?